packages/components/molecules/f-rating/src/components/_tests/Rating.test.js

```diff
@@ -211,6 +211,72 @@ describe('Rating', () => {
                 });
             });
         });
+
+        describe('`shouldDisplayUserOwnRating`', () => {
+            it('should exist', () => {
+                // Arrange
+                propsData.starRating = 2;
+                wrapper = shallowMount(VRating, {
+                    propsData,
+                    localVue,
+                    i18n
+                });
+
+                // Act & Assert
+                expect(wrapper.vm.shouldDisplayUserOwnRating).toBeDefined();
+            });
+
+            describe('when invoked', () => {
+                describe('and `isUserRating` is falsey and reviewCount is truthy', () => {
+                    it('should return false', () => {
+                        // Arrange
+                        propsData.starRating = 2;
+                        propsData.reviewCount = 100;
+                        wrapper = shallowMount(VRating, {
+                            propsData,
+                            localVue,
+                            i18n
+                        });
+
+                        // Act & Assert
+                        expect(wrapper.vm.shouldDisplayUserOwnRating).toBe(false);
+                    });
+                });
+
+                describe('and `hasRatingAvailable` is falsey and `isUserRating` is truthy', () => {
+                    it('should return false', () => {
+                        // Arrange
+                        propsData.starRating = 2;
```

Comment:
I don't think it makes sense to set the star rating if there are no reviews available

Reply:
I will need to set it to zero as it is a required prop.